src/components/Feed.js

```diff
@@ -3,16 +3,18 @@ import { useEffect, useState } from 'preact/hooks'
 import { ListView } from 'components'
 import { useI18n } from 'hooks'
 import { getTrendingArticles } from 'api'
+import { getUserCountry } from 'utils'
 
 export const Feed = ({ lang, isExpanded, setIsExpanded, lastIndex, setNavigation, containerRef }) => {
   const [trendingArticles, setTrendingArticles] = useState([])
   const i18n = useI18n()
+  const userCountry = getUserCountry()
 
   useEffect(() => {
-    const [request, abort] = getTrendingArticles(lang)
+    const [request, abort] = getTrendingArticles(lang, userCountry)
     request.then(setTrendingArticles)
     return abort
-  })
+  }, [lang, userCountry])
```

Comment:
Just wondering do we need these 2 state here? We will never reach here if one of them is null

Reply:
That's correct, realized that when working on loading state PR (https://github.com/wikimedia/wikipedia-kaios/pull/337/files#diff-ed6756f61e75bd6003f7ec1a246fa0444f9d4074a95da69e536e5ee39b6a09a4R20)